src/infrastructure/dataSchemas/allDataSchema.ts

```diff
@@ -0,0 +1,10 @@
+import { SemanticVersion } from '@/@types/types'
+import { configDataSchema } from './configDataSchema'
+import { PlotDataSchema } from './plotDataSchema'
+
+export interface allDataSchema {
+  appVersion: SemanticVersion
+  plotImageBase64: string
```

Comment:
Added plot image as base 64 string.
The reason why it is not in the plotDataSchema is that we may want to export plot data without image data.